website/docs/quickstart-guide/systemd.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 4
+---
+
+# Systemd Unit
+
+With both DEB and RPM package we ship the systemd unit, to start FerretDB automatically.
+If the FerretDB is not installed yet, please refer to the one of the matching pages.
+
+The unit file provides some simple environment variables as an example. 
+They should be overwritten with the proper configuration, with the [correct flags](../configuration/flags.md).
+
+```
+[Service]
+ExecStart=/usr/bin/ferretdb
+Restart=on-failure
+
+# Configure the FerretDB service with `systemctl edit ferretdb`.
+# For more configuration options check https://docs.ferretdb.io/configuration/flags/
+
+Environment="FERRETDB_POSTGRESQL_URL=postgres://username:password@127.0.0.1:5432/ferretdb"
+Environment="FERRETDB_LISTEN_ADDR=:27017"
+Environment="FERRETDB_DEBUG_ADDR=:8088"
+```
+
+You can modify them by using systemctl edit ferretdb command. This'll open a text editor that'll allow you to
+override the systemd options of choice.
+For example, if we want to use SQLite backend instead of PostgreSQL, we could write something like:
+
+```
```

Comment:
what's the file format for this? "```text"?

Reply:
But it should be added to `additionalLanguages` in Docusaurus config